pytest_celery/message_brokers/message_broker.py

```diff
@@ -27,7 +27,7 @@ def start(self) -> None:
     def stop(self) -> None:
         """"""
         self.container.stop()
-        self.healthcheck_scheduler.stop()
+        self.healthcheck_scheduler.shutdown()
```

Comment:
It appears we need an [ExitStack](https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack).